### ISSUE TITLE
Support for dynamic loading ?

### ISSUE THREAD

**Issue by EricGrange** [neslib/Chet#38]
Current headers are converted to static bindings (with optional "delayed").

Would it be possible to have an option to generate dynamic bindings instead ? (with GetProcAddress, etc)

The use case would not be just to survive a dll not being present, but also allow the dll name and path to be specified.

**Comment by jarroddavis68** [comment]
Click the post process button and click the help. I added a PP command to do just that. I hope to submit a PR at some point with all my fixes and enhancements since the last update. Sorting through some corner cases.

**Comment by EricGrange** [comment]
@jarroddavis68 yes, that's one way I was looking at it (but doing it directly in small Delphi executable)
Did you publish your PP command ?

**Comment by jarroddavis68** [comment]
![image](https://github.com/user-attachments/assets/424fb992-6d9a-411f-9b0a-903c8bf39057)
`CreateDynamicImport(0)`
this post processing command will create:
![image](https://github.com/user-attachments/assets/ecc42fdc-fde3-4089-8963-aded9719a83a)

![image](https://github.com/user-attachments/assets/09357980-f1bc-4a41-8f64-ed0907318907)

then you can call `GetExports` in your own code to dynamically load. This is in now. You will run into a few issues that a future PR will address, but it does work.

**Comment by EricGrange** [comment]
Thanks!

FWIW use case is for webgpu.h (https://github.com/EricGrange/Delphi-WebGPU) where you have several implementations, and each implementation regularly churns out builds, so plenty of "compatible" dlls to choose from :)

**Comment by EricGrange** [comment]
Hmm, the "CreateDynamicImport(0)" in Post process doesn't seem to have any effect ?
Though I'm not even sure the post process script is executed at all (a lone SaveToFile doesn't create anything  AFAICT and there is no error when placing garbage in the pp script)

**Comment by jarroddavis68** [comment]
post processing will effect this output:
![image](https://github.com/user-attachments/assets/bac4ce36-67fb-4b96-88ff-9d2885f0ad1f)

here is a build with all the PP changes I've made since the release that you have:
[Chet.zip](https://github.com/user-attachments/files/16982345/Chet.zip) | [VirusTotal Report](https://www.virustotal.com/gui/file/63655f1c0724f0af1591d9e3652826d7efe03ded11f7f4a24d67bbdd9c088b48?nocache=1)

So, the script processing is basic, enough to get the job done. If the script command is mispelled and/or not in the correct format, it will just be silently ignored. For example, if you add a `;` at the end, like I tend to do at times, that command will be ignored. So, if it does not seem to be working, check the syntax and make sure it's correct.

**Comment by EricGrange** [comment]
@jarroddavis68  with your build this worked, thanks.

As a small side-request, maybe the post-processing should appear in the translate log ? Currently it's not mentioned, and if the PP script contains invalid commands, there is no error reported either.
